### DISCUSSION TITLE
Memory Leak?

### DISCUSSION THREAD

**Discussion by bobba84** [Chia-Network/chia-blockchain#8447] [Support]
Hello, my start_full_node consumes 20GB+ RAM after a few days. I've trie4d reinstalling twice, even deleting the whole app folder. I am on latest version. Thanks!

**Comment by loppefaaret** [comment]
there is another report of a memory leak here: #8442

**Comment by BrandtH22** [comment]
@bobba84 , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
